website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,306 @@
+
+## Migrate from _Prettier_ with a single command
+
+[_Biome v1.6_ introduced the command `biome migrate prettier`](/blog/biome-v1-6/#easier-migration-from-prettier).
+
+This new version adds the support of the `overrides` field and attempts to convert `.prettierignore` glob patterns to globs supported by _Biome_.
+
+Given the following `.prettierrc.json`
+
+```json title=".prettierrc.json"
+{
+	"useTabs": false,
+	"singleQuote": true,
+	"overrides": [
+		{
+      		"files": ["*.json"],
+      		"options": { "tabWidth": 2 }
+    	}
+	]
+}
+```
+
+Run `biome migrate prettier --write` to migrate your _Prettier_ configuration to _Biome_.
+This results in the following _Biome_ configuration:
+
+```json title="biome.json"
+{
+	"formatter": {
+		"enabled": true,
+		"formatWithErrors": false,
+		"indentStyle": "space",
+		"indentWidth": 2,
+		"lineEnding": "lf",
+		"lineWidth": 80,
+		"attributePosition": "auto"
+	},
+	"organizeImports": { "enabled": true },
+	"linter": { "enabled": true, "rules": { "recommended": true } },
+	"javascript": {
+		"formatter": {
+			"jsxQuoteStyle": "double",
+			"quoteProperties": "asNeeded",
+			"trailingComma": "all",
+			"semicolons": "asNeeded",
+			"arrowParentheses": "always",
+			"bracketSpacing": true,
+			"bracketSameLine": false,
+			"quoteStyle": "single",
+			"attributePosition": "auto"
+		}
+	},
+	"overrides": [
+		{
+			"include": ["*.json"],
+			"formatter": {
+				"indentWidth": 2
+			}
+		}
+	]
+}
+```
+
+Note that this overrides your initial _Biome_ configuration.
+The command requires _Node.js_ to be installed to load _JavaScript_ configurations such as `.prettierrc.js`.
+For now, `biome migrate eslint` doesn't support configuration written in _JSON5_, _TOML_, or _YAML_.
+
+
+## Emit formatting and linting reports
+
+_Biome_ is now able to output _JSON_ reports detailing the diagnostics obtained from a run.
+
+For instance, you can emit a report when you lint a codebase:
+
+```shell
+biome lint --reporter=json-pretty .
+```
+
+For now, we support two report formats: `json` and `json-pretty`.
+
+Note that the report format is subject to breaking changes, so you **should not** rely on it yet.
+Please try this feature and let us know if any information is missing from the reports.
+
+
+## Check _Git_ staged files
+
+_Biome v1.5_ added the `--changed` flag on its main commands `biome format`, `biome lint`, and `biome check` to format and lint _Git_ tracked files that have been changed.
+
+Today we are introducing a new flag `--staged` which allows you to check only files that have been added to the _Git index_ (_staged files_).
+This is useful for checking that the files you want to commit are formatted and linted:
+
+```shell
+biome check --staged .
+```
+
+This is handy for writing your own [pre-commit script](/recipes/git-hooks/#shell-script).
+Note that, unstaged changes on a staged file are **not** ignored.
+Thus, we still recommend using a [dedicated pre-commit tool](/recipes/git-hooks/).
+
```

Comment:
`--changed` is available in CI and people use it.

Also, saying "don't make sense" is a bit reductive, and you should explain why, so beginners can understand why.